test/uk/gov/hmrc/trusts/utils/DomainValidatorSpec.scala

```diff
@@ -74,5 +75,101 @@ class DomainValidatorSpec extends BaseSpec with DataExamples {
     }
   }
 
+  "trusteesDobIsNotFutureDate" should {
+    "return None when there is no trustees" in {
+      val request = registrationWithTrustess(None)
+      SUT(request).indTrusteesDobIsNotFutureDate.flatten.isEmpty mustBe true
```

Comment:
Can this assertion must be `SUT(request).indTrusteesDobIsNotFutureData must not be defined` or `must be empty`